extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -113,21 +97,46 @@ export const SendAmount = ({ previous }: { previous: ROUTES }) => {
     dispatch(saveAsset(values.asset));
     if (values.destinationAsset) {
       dispatch(saveDestinationAsset(values.destinationAsset));
-      dispatch(saveConversionRate(conversionRate));
     }
     navigateTo(ROUTES.sendPaymentSettings);
   };
 
-  const handleAssetSelect = (e: React.ChangeEvent<HTMLSelectElement>) => {
-    const selected = e.target.value;
-    if (accountBalances.balances) {
-      setAssetInfo({
-        code: accountBalances.balances[selected].token.code,
-        balance: accountBalances.balances[selected].total.toString(),
-        canonical: selected,
-      });
-    }
-  };
+  const formik = useFormik({
```

Comment:
I experimented in using `useFormikContext` and wrapping the majority of the component in another one. Got pretty messy/unnecessarily complex with data management (having to pass data down to another component or pulling from redux twice).

I think useFormik makes sense for now since it's simpler, if we find we need the formik context stuff later on we could change then (and then possibly change the SendTo component as well)